python/docs/hashtable_repeated_word/README.md

```diff
@@ -0,0 +1,41 @@
+# Challenge Summary
+Find repeated words
+
+## Whiteboard Process
+[whiteboard](python/docs/hashtable_repeated_word/whiteboard.jpeg)
+
+## Approach & Efficiency
+O(n) since we iterate through all the words until we find a match.
+This means we potentially need to check every word in the words list, if the
+last word is our positive match.
```

Comment:
This is good! I recommend breaking up the time and space complexity analysis and touching on each individually.